README.md

```diff
@@ -3,7 +3,6 @@
 [![npm version](https://img.shields.io/npm/v/@core-ds/primitives.svg?style=flat-square)](https://www.npmjs.com/package/@core-ds/primitives)
 
 - [Reference](https://unpkg.com/@core-ds/primitives)
-- [Figma](https://www.figma.com/file/3HaqcNx1TsdkyGAJqHGzZW/Core-Primitives)
```

Comment:
were not using figma anymore?

Reply:
Oh, my bad - I forgot to mention why I removed that. The Figma will be obsolete once we move to the v2.x.x colors